dll/appcompat/shims/genral/ignorefreelib.c

```diff
@@ -114,15 +114,12 @@ static VOID InitIgnoreFreeLibrary(PCSTR CommandLine)
     return;
 
 fail:
-    --n;
-    while (n >= 0)
+    while (--n >= 0)
```

Comment:
Not an improvement

Reply:
Reverted.
Although I wonder why spreading a "simple" loop over separated lines is better than a one-liner `while()` or `for()`.
NB: This file has 3 times a similar pattern.

Reply:
What's "better" is not changing code that's not broken, unless you're the maintainer of the module.